core/drivers/stpmic1.c

```diff
@@ -721,10 +721,11 @@ int stpmic1_bo_voltage_unpg(struct stpmic1_bo_cfg *cfg)
 	if (stpmic1_register_read(cfg->ctrl_reg, &value))
 		return -1;
 
-	value &= cfg->mask;
-	value = MIN(value, cfg->min_value);
+	if (value & cfg->mask < cfg->min_value)
+		return 0;
```

Comment:
The code looks sensible to me, now I think the commit description sounds weird: "stpmic1_bo_voltage_unpg() that applies boot-on voltage is updated to not lower current voltage if above min voltage constraint." => do you mean "stpmic1_bo_voltage_unpg() that applies boot-on voltage is updated to not lower current voltage **below** min voltage constraint"? (I really don't get the "if above..." part)

Reply:
> if masked voltage is higher than min voltage:
> then nothing to do;
> else force update of regulator to min voltage;

Ah so the function is checking and possibly correcting the voltage. So be it ;-)
Perhaps `>=` because if equal there is nothing to do either.

Reply:
yes `>=`. will fix.